ProductsManegements/Product.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SimpleInventoryManagementSystem.ProductsManegements
```

Comment:
can we rename the namespace for ProductsManagement instead of ProductsManagement**s**